monkey/infection_monkey/propagation_credentials_repository/propagation_credentials_repository.py

```diff
@@ -25,14 +26,15 @@ class PropagationCredentialsRepository(IPropagationCredentialsRepository):
     def __init__(
         self,
         island_api_client: IIslandAPIClient,
+        manager: SyncManager,
         polling_period: float = CREDENTIALS_POLL_PERIOD_SEC,
     ):
         self._island_api_client = island_api_client
         self._polling_period = polling_period
         context = get_context("spawn")
         self._lock = context.Lock()
         self._next_update_time = context.Value("d", 0, lock=False)
```

Comment:
Same question. `manager.Lock()` and `manager.Value()`?

Reply:
Maybe it's not necessary and would be less performant? It might be worth dropping a comment in with details whenever we reach a decision.

Reply:
After some discussion, we'll just leave this as is.